assets/javascripts/discourse/templates/components/resolution-list-item.hbs

```diff
@@ -5,27 +5,10 @@
         {{i18n "communitarian.resolution.list.date" period=formattedActionPeriod}}
       {{/if}}
     </span>
-    <span>
-      {{#if openedPoll}}
-        {{i18n "communitarian.resolution.list.votes" voters=openedPoll.voters}}
-      {{/if}}
-    </span>
   </div>
   <p class="resolution-list-item__name">
     {{~topic-link resolution class="raw-link raw-topic-link"}}
   </p>
-  <ul class="resolution-list-item__options-list">
-    {{#each openedPoll.options as |poll_option|}}
-      <li class="resolution-list-item__option">
-        <label for="votes" class="resolution-list-item__option-name">
-          {{poll_option.html}}
-        </label>
-        <div class="resolution-list-item__option-value-wrap">
-          <div class="resolution-list-item__option-value {{if (compare poll_option.votes '===' mostPopularOption) "max"}}"
-              style={{get-percent-width poll_option.votes mostPopularOption}}>
-          </div>
-        </div>
-      </li>
-    {{/each}}
-  </ul>
+
+  {{mount-widget widget="communitarian-resolution-widget" args=resolution}}
```

Comment:
It seems like all works fine, but we need to use our design instead of default for voted resolution view:
default design:
<img width="702" alt="Снимок экрана 2020-09-16 в 19 06 44" src="https://user-images.githubusercontent.com/6059005/93363476-f84b2300-f84f-11ea-8297-5de97a793ed2.png">
Our design:
<img width="705" alt="Снимок экрана 2020-09-16 в 19 06 22" src="https://user-images.githubusercontent.com/6059005/93363484-fb461380-f84f-11ea-9552-bd627ca94920.png">